ot/lp/network_simplex_simple.h

```diff
@@ -1507,7 +1508,7 @@ namespace lemon {
 			if( retVal == OPTIMAL){
                 for (int e = _search_arc_num; e != _all_arc_num; ++e) {
                     if (_flow[e] != 0){
-                        if (abs(_flow[e]) > EPSILON)
+                        if (fabs(_flow[e]) > _EPSILON)
```

Comment:
maybe add a comment to say that it was switched to _EPSILON due to numerical problems and refer to the Issue Number